share/sysbuild/cmake/modules/sysbuild_extensions.cmake

```diff
@@ -106,24 +106,27 @@ endfunction()
 #   ExternalZephyrProject_Add(APPLICATION <name>
 #                             SOURCE_DIR <dir>
 #                             [BOARD <board>]
+#                             [BOARD_REVISION]
```

Comment:
missing `<revision>` here, and maybe group it inside `BOARD` so that it's clearer this argument is only valid together with BOARD.

For example:
```suggestion
#                             [BOARD <board> [BOARD_REVISION <revision>]]
```

minimal change requested:
```suggestion
#                             [BOARD <board>]
#                             [BOARD_REVISION <revision>]
```

Reply:
Fixed.